routes/comments.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
```

Comment:
I would prefer the file name be comments_routes.
But it you think it's better this way I'll change mine